### DISCUSSION TITLE
How to use jax.random package for sampling general normal distribution?

### DISCUSSION THREAD

**Discussion by EngineerKhan** [jax-ml/jax#8149] [Q&A] [answered]
While `jax.random.normal()` gives standard normal distribution. How can we use the jax.random package to sample a non-standard normal distribution (without using the shifting property, etc.) like we do [in Numpy](https://numpy.org/devdocs/reference/random/generated/numpy.random.normal.html)?

**Comment by jakevdp** [comment, marked as answer]
The equivalent of
```python
x = np.random.normal(mean, std, shape)
```
would be
```python
x = mean + std * jax.random.normal(key, shape)
```
